### PR TITLE
Add linter

### PR DESCRIPTION
**Pull request by tbatard**
Initial commit to follow steps described in: https://hackmd.io/l4yoHHGRRBC2yi4pQ7Q0OA?view

### CODE REVIEW

File: hack/lint.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright 2017 the Velero contributors.
```
[the remark this answers is not in the document]
Comment (tbatard) [reply hack/lint.sh]: Thanks, it should be good now

**Comment by nrb** [comment]
@tbatard Looks like there's a merge conflict, as we updated some of the logic on the Makefile to account for git submodules.

**Comment by tbatard** [comment]
@nrb Just rebased, there should be no more conflicts

**Comment by nrb** [comment]
@tbatard Thanks! Do you still consider this WIP, or is it ready for review?

**Comment by tbatard** [comment]
Hi @carlisia. It's not yet ready to merge. We need to add it part of the GH workflow. Once it's done we'll remove the WIP from the title.
Thank you for all your feedbacks!

**Comment by tbatard** [comment]
@carlisia we're ready to merge.
We've added the github action on PR to run the linter on the current changes and the documentation related to it

**Pull request merged**